sections/5_reproducibility/reproducibility_composite_confidence_index.qmd

```diff
@@ -0,0 +1,249 @@
+
+- **Scopus**  
+  [Scopus](https://www.scopus.com/) is a large citation database that includes the Field-Weighted Citation Impact (FWCI) indicator. It can serve as a source for citation data and normalised impact values used in RCCI.  
+
+- **Web of Science / InCites**  
+  [Web of Science](https://webofscience.com/) provides citation data and normalised citation metrics through InCites, where the Category Normalised Citation Impact (CNCI) is implemented. This can be used as an alternative to Scopus FWCI or Dimensions FCR.  
+
+- **DataCite**  
+  [DataCite](https://datacite.org/) is a registry that provides persistent identifiers (DOIs) and metadata for research datasets and software. It is especially useful for retrieving metadata elements (Name, Version, License, URL) needed for calculating the FAIR Index.  
+
+- **Crossref**  
+  [Crossref](https://www.crossref.org/) maintains extensive metadata for scholarly publications and related outputs, including references and links to datasets and software. It is valuable both for reuse tracking (FWRI) and FAIR metadata extraction (FI).  
+
+- **Zenodo / Figshare / Institutional Repositories**  
+  These repositories host datasets, software, and other artefacts. They expose metadata via APIs, which can be used to evaluate FAIRness and retrieve usage information for reuse analysis.  
+
+- **scite.ai**  
+  [scite.ai](https://scite.ai/) provides classification of citation statements into supporting, refuting, or mentioning. It can be used to measure polarity of publications and calculate the RCI.  
+
+---
+
+# Existing Methodologies
+
+## SciNoBo Toolkit
+
+The [SciNoBo Toolkit](https://scinobo.ilsp.gr/toolkit) has implemented and operationalised the RCCI and its component indicators into a **working monitoring dashboard**.  
+
+- In the **TIER2 project**, SciNoBo was used to extract artefacts from project deliverables and publications, link them to citation and reuse data, and compute FWCI, FWRI, FI, RCI, and RCCI.  
+- The RCCI results were presented in **pilot dashboards** for funders and RPOs.  
+- The approach was validated and refined through **stakeholder feedback** in webinars and presentations.  
+
+This makes RCCI not only a conceptual indicator, but also one that has been **implemented and tested in practice**.
+
+---
+
+## Other methodologies
+
+While SciNoBo currently offers the most complete implementation, other methodologies and tools can be used to compute individual RCCI components:  
+
+- **Citation normalisation**  
+  FWCI can be derived using normalisation approaches described in the [Citation Impact](../2_academic_impact/citation_impact.qmd) indicator, based on expected citation counts per field and year. This methodology is implemented in bibliometric databases such as Web of Science/InCites (CNCI), Scopus (FWCI), and Dimensions (FCR).  
```

Comment:
Most notably, it is also implemented in OpenAlex, which should also be included given its open character.